sql/test_create.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+docker rm -f devtest;\
+docker run --name devtest --mount type=bind,source="$(pwd)",target=/sql -d postgres:alpine;\
+sleep 5;\
+docker exec -it devtest psql -U postgres -c "CREATE DATABASE testdb;";\
+docker exec -it devtest psql -U postgres -f /sql/S1_CREATE_DOMAINS.sql;\
+docker exec -it devtest psql -U postgres -f /sql/S2_CREATE_USER_TABLE.sql;\
+docker exec -it devtest psql -U postgres -f /sql/S3_CREATE_DOCUMENT_TABLE.sql;\
+docker exec -it devtest psql -U postgres -f /sql/S4_CREATE_SHARED_TABLE.sql;\
```

Comment:
The SQL scripts in dev are the same sql scripts to be used in production, therefore DB management and versioning is important.

There are several ways of versioning sql scripts and DB migration. These are the examples:
 - https://bitbucket.org/liamstask/goose/
 - https://github.com/golang-migrate/migrate guide -> https://github.com/golang-migrate/migrate/blob/master/MIGRATIONS.md
- https://github.com/robinjoseph08/go-pg-migrations

I am open to any of these three.